src/test/lib/storage/fixed_string_dictionary_segment/fixed_string_test.cpp

```diff
@@ -112,6 +112,15 @@ TEST_F(FixedStringTest, Swap) {
   EXPECT_EQ(fixed_string, "foo");
 }
 
+TEST_F(FixedStringTest, Swap) {
+  std::vector<char> char_vector = {'b', 'a', 'r'};
+  FixedString fixed_string3 = FixedString(&char_vector[0], 3u);
```

Comment:
`auto` * 2. `char_vector` could be `const`, as well. Please remove the unsigned literal if possible, e.g., 
```suggestion
  const auto char_vector = std::vector<char>{'b', 'a', 'r'};
  auto fixed_string3 = FixedString(&char_vector[0], 3);
```